tests/e2e-cypress/support/commands.js

```diff
@@ -344,3 +344,22 @@ Cypress.Commands.add('addProfileJsonFixture', (mockupData) => {
         }).as('profile')
     }
 })
+
+function cesiumTilesLoaded(viewer) {
+    return new Cypress.Promise((resolve) => {
+        viewer.scene.postRender.addEventListener(() => {
+            if (viewer.scene.globe.tilesLoaded) {
+                resolve()
+            }
+        })
+    })
+}
+
+Cypress.Commands.add('waitUntilCesiumTilesLoaded', () => {
+    cy.readWindowValue('cesiumViewer').then((viewer) => {
+        cy.wrap(null).then({ timeout: 50000 }, async () => {
+            await cesiumTilesLoaded(viewer)
+            return viewer
+        })
```

Comment:
might be good idea to add this comment you were suggesting, about this blog post describing this `cy.wrap(null)`